### ISSUE TITLE
Discord Bot and Discord Bot Routes Semantics - Child of #5290

### ISSUE THREAD

**Issue by timolegros** [hicommonwealth/commonwealth#5332]
Child of #5290

## Description

The `discordConsumer.ts` contains the following references to the `Chains` table:
- A raw query to fetch topics -> will be handled in #5330 NOT THIS TICKET
- [ ] `/api/bot/threads` route body property `chain`
- [ ] `/api/bot/threads/${thread_id}/comments` route body property `chain`
- [ ] `/api/bot/comments/${parsedMessage.message_id}` route body property `chain`

## Project Owner

@timolegros

## Engineering Requirements

- Updated route body properties

**Comment by rbennettcw** [comment]
Already done.

**Comment by timolegros** [comment]
@rbennettcw I do not think this is complete. From what I can see we _are_ using `chain` rather than `community_id` in post requests from the Discord bot. This is handled by the `validateCommunityWithTopics` middleware which looks for `req.chain` _and_ `req.community`.
